node/index.js

```diff
@@ -0,0 +1,52 @@
+const { Pool } = require('pg')
```

Comment:
I would suggest running the file through prettier, e.g. [as so](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAExMAUIQA2mAvpgLyYBOcAjgK4CWNAFAOQAOA5mwJQA6UQYThYOBQoMGQoGTIyiMYAFQCGAIxEVMqtAE8oYTCw68KAPmyDMmGXLCFGCLJVUB3VUswcAdDKhwYDAsAlA2dkRwPoQQXOwAkoowjKqOAF4KXJgwGiI++XzW2VR6VmE2tuhYOZpwiUop6XAAJtrunlgOTrA+DHAlLEUVmAAGAMIASgCiAILKU5jKMwBCADIL8QBimAByAPLKmFMAGvEAyspnOmCQ9LBoxkPDFYytAKpv8QAieBPxALIzCYATUwAGkpqCvlNNjM3qtDlwEAB9KiqKDNCAAW2R9CYzRCABons9MOpUuiwHBMPEdsoABwkiq8EZPULDCJ5GJxGoierJVKMNItdmkWyqGBgAAWxn6VDMwCenKi3JYcp8GFUYAA1qKyAAzBSpQilRXlcKOZw+GgiXRwEJFEiCJ1CQxVeSyfowGY3CB3GAPVz6QzGUwWMrhd1dZxtDxeXz+QLBUXK6KxBKeqjJKBZAJuagQNwPBQwCDZKXUrW3e7ZXJRAop93kwiUuAANVS9GplAA2v8JVKfPqYhAqCx+zBB2iMdiQpgAFSYACMAAY17wALpFGAlCMVOxYKt++6JNBelqxjq2S09PoDJlsWlnKYTQ605R7a7VgPGV6EskUoYcBmG2MyrG8UxXCw0KwvCyj-gAJEuvBsMS5oVM2rYdoQXZskq6CRGmcRHv6aCnuezR6uKkoymqVDynukayIRqrqpqOpUYaUDGqaSo3jA1pwLaZ4OuULouoIcAAB7iFmaA+FK6LNCIVBtMGRgsDI8BSTAZjkJYZo2Iw+rGAAhOIRAKvhsidNAARBIw0BnDumTaBwVAQFSaDyQgABuPhfHMKwzM+yJvBMqxPBZxCUHmeASCwhnPImDlOS5OZoaSWKqFJSDLplzzZVJqzGaIjBYnAZyBNAzRoHlS70muTwkHqUjlO0XgKEoai1CYEiih1WAKGeWY+t+aB9ZZzogISIAQBwyToMgoCqPRha4KtCBoMgICpB4ejbbN6hojqohnKoFUlQEyD6qkZ5HSd2pnRwWqZMgO5drNcBYuoLTNC0qzolw9CqEimyjtlMDZlwO2qPQpYzSAUowFihAAOpSkocBoC9VJVbIDS+Uoeg7WA3mI8NXq4GiXDZTdd1wLNABWaBSWcmQiAAivQEDwPThD3SAL1UCNO3kr9hCI+5JZo68k7II1s3uRAZ5o2iHA7e52P9L5cCIwwvNwNT83bSgugALQBC0LSIzQDDMEbNN00gt0C4zIBnlijDvVQn0exzcDc4b-OCzUsvNPLSAAEyzTuniODmYzYs7IDYwArIj9Bnj1puu4Lvldok-2wGcYBUIwC0zBizl6CIIdwCQJBAA). I think the only difference is that it adds semicolons, but I did notice the absence of semicolons 😛.

Reply:
Good idea. I was just using eslint, but it looks like my settings didn't catch the semicolons. See https://github.com/cockroachlabs/examples-aws-lambda/pull/6.